CONTRIBUTOR.md

```diff
@@ -0,0 +1,124 @@
+# Contributor guide
+
+#### Read guide below if you would like to know how to run this application in your local environment to test out features you'd like to implement!
+
+##### Note that this file walks you through using Github Desktop + Visual Studio Code
+
+### Quick access index
+
+[Step 1](#step-1)
```

Comment:
It's odd the link isn't working in the file diff preview, but if you click on "view file" you can see it. It still doesn't take you anywhere though, because the link has to match the full title slug. You can inspect a title with your web console to get the actual href value.

```suggestion
[Step 1](#step-1---clone-and-open-in-visual-studio-code)
```

Reply:
Ah ok, will make those changes once i get backn
 online. Thanks for the tip and i'll also check inspect elementm